feature-flags/src/api.rs

```diff
@@ -0,0 +1,58 @@
+use std::collections::HashMap;
+
+use axum::http::StatusCode;
+use axum::response::{IntoResponse, Response};
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+#[derive(Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub enum FlagsResponseCode {
+    Ok = 1,
+}
+
+#[derive(Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct FlagsResponse {
+    pub error_while_computing_flags: bool,
```

Comment:
.. to give client more info about the error?

This would usually only be a db error, so internal, not very useful to expose to clients.

For now I want the same shape as the existing endpoint, so will keep this as is